<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>telaCadastroListaFilme</class>
 <widget class="QWidget" name="telaCadastroListaFilme">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Build a video file list for blinding procedures</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>90</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resourcefile.qrc">:/icons/logo_ethowatcher.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Video Info</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QWidget" name="widget_6" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QWidget" name="widget_7" native="true">
                <property name="minimumSize">
                 <size>
                  <width>400</width>
                  <height>0</height>
                 </size>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <widget class="QWidget" name="widget" native="true">
                   <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                     <widget class="QLabel" name="label">
                      <property name="text">
                       <string>Name</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="lblNome">
                      <property name="minimumSize">
                       <size>
                        <width>300</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="autoFillBackground">
                       <bool>true</bool>
                      </property>
                      <property name="text">
                       <string/>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QWidget" name="widget_2" native="true">
                   <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <item>
                     <widget class="QLabel" name="label_4">
                      <property name="text">
                       <string>Path</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="lblCaminho">
                      <property name="minimumSize">
                       <size>
                        <width>300</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="autoFillBackground">
                       <bool>true</bool>
                      </property>
                      <property name="text">
                       <string/>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QWidget" name="widget_3" native="true">
                   <layout class="QHBoxLayout" name="horizontalLayout_3">
                    <item>
                     <widget class="QLabel" name="label_6">
                      <property name="text">
                       <string>Extension</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="lblExtensao">
                      <property name="minimumSize">
                       <size>
                        <width>300</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="autoFillBackground">
                       <bool>true</bool>
                      </property>
                      <property name="text">
                       <string/>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QListWidget" name="listWidget"/>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget_8" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_5"/>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_4" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPushButton" name="pbOpenXml">
                <property name="text">
                 <string>Open registered video</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget_9" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <item>
                  <widget class="QPushButton" name="pbAddVideo">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="text">
                    <string>Add Video to list</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pbDeletar">
                   <property name="text">
                    <string>Delete</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbGravarXML">
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>240</y>
         <width>151</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Criar nova lista de videos</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>240</y>
         <width>151</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Gerar Videos com pluco cego </string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Calibri'; font-size:12pt; color:#000000;&quot;&gt;Find help, instructions and a standard operating procedure in&lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;: &lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;https://ethowatcher.com&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resourcefile.qrc"/>
 </resources>
 <connections/>
</ui>
